### PR TITLE
Disable root and jailbreaks checks, present message to show they are disabled

### PR DESCRIPTION
**Pull request by shffrror**
It looks like the IDE did some formatting changes, let me know if we want to change those.

I've flagged the actual lines with the changes for ease.

### CODE REVIEW

File: phone_scanner.py [hunk 1]
```diff
@@ -333,68 +344,77 @@ def app_details(self, serialno, appid):
...
         #        (hf_recent['timestamp'].map(str) if isinstance(hf_recent['timestamp'], datetime) else 'nooo') +")"
-        #d['recent_permissions'] = hf_recent['timestamp']
-        #print(d['recent_permissions'])
+        # d['recent_permissions'] = hf_recent['timestamp']
+        # print(d['recent_permissions'])
         return d, info
 
     def isrooted(self, serial):
+        # TODO This should be removed once the check is fixed
+        return (False, "Jailbreak and root checks are currently disabled")
```
Comment (shffrror) [inline phone_scanner.py:378]: First change here

File: phone_scanner.py [hunk 2]
```diff
@@ -505,43 +529,50 @@ def _dump_phone(self, serial):
...
-            else:
-                rooted['True'].append("Filesystem *might* be rooted. Conduct additional checks.")
-        except FileNotFoundError as e:
-            print("Couldn't find Jailbroken FS check log.")
-            # TODO: trigger error message? like 
-            # TODO: show a try again, maybe it's not plugged in properly. still not working? this could be due to many many many reasons.
-            #return (True, ['FS check failed, jailbreak not necessarily occurring.'])
+
+        # TODO This should be removed once the check is fixed
+        rooted['False'].append("Jailbreak and root checks are currently "
```
Comment (shffrror) [inline phone_scanner.py:553]: Second change here

**Comment by rchatterjee** [comment]
Yeah, the IDE did some changes, some of which are good. But for this PR,  let's just do the necessary line changes, and save the linting changes to a different PR.

**Comment by shffrror** [comment]
> Yeah, the IDE did some changes, some of which are good. But for this PR, let's just do the necessary line changes, and save the linting changes to a different PR.

Done!

**Review by rchatterjee — APPROVED** [review]

**Pull request merged**